ci/test/pipeline.template.yml

```diff
@@ -290,6 +290,7 @@ steps:
     plugins:
       - ./ci/plugins/mzcompose:
           composition: restart
+          run: stash
```

Comment:
@benesch are you sure about that change? It will cause only 1 out of several tests in test/restart to run ?

Reply:
😱 not at all! Thanks so much for double checking.

Reply:
https://github.com/MaterializeInc/materialize/pull/16856